CHANGELOG.D/3137.feature

```diff
@@ -0,0 +1,12 @@
+Balance is no longer stored on a cluster level, and was moved to an organization level, e.g.,
+to an org itself, and to an org users, instead of a cluster / cluster users.
+
+New commands:
+  - `set_org_defaults` - allows to set an organization default values, such as a default user credits
```

Comment:
```suggestion
  - `apolo admin set-org-defaults` - allows to set an organization default values, such as a default user credits
```
`click` replaces underscores with dashes.
The full command path is more informative from end user point of view maybe, but the final decision is up to you.

The same is true for other mentioned commands.

Reply:
@asvetlov changed to a full commands everywhere and used a dashes. Thanks!